apis/facebook/api/src/facebook_i.e

```diff
@@ -60,6 +60,22 @@ feature --Get - User
 		deferred
 		end
 
+	user_likes (a_user_id: STRING; a_params: detachable FB_PAGE_PARAMETER): detachable FB_EDGES [FB_PAGE]
+			-- All the Pages this person has liked
+			-- GET /{user_id}/likes	
+		note
+			EIS: "name=User likes", "src=https://developers.facebook.com/docs/graph-api/reference/user/likes/", "protocol=uri"
+		deferred
+		end
+
+	user_groups (a_user_id: STRING; a_params: detachable FB_GROUP_PARAMETER): detachable FB_EDGES [FB_GROUP]
+			-- The Facebook Groups that the person is an admin of
+			-- GET /{user_id}/groups	
+		note
+			EIS: "name=User Groups", "src=https://developers.facebook.com/docs/graph-api/reference/user/groups/", "protocol=uri"
+		deferred
+		end
+
 feature	-- Post
 
 	post (a_post_id: READABLE_STRING_32; a_params: detachable FB_POST_PARAMETER): detachable FB_POST
```

Comment:
comments?

Reply:
It has the following comment:
			-- An individual entry in a profile's feed. The profile could be a user, page, app, or group.
			-- GET /{a_post_id}